### PR TITLE
1.5.6

### PR DESCRIPTION
**Pull request by boiscljo**
Follow-up to https://github.com/pronze/SBA/pull/216 applying all changes requested with correct version number and without any eclipse configuration file mixed in. Used a clean branch to prevent further development to be mixed in before the actual merging.

### CODE REVIEW

File: api/.project [hunk 1]
```diff
@@ -5,19 +5,13 @@
 	<projects>
```
Comment (zlataovce) [inline api/.project]: remove this file entirely

Comment (boiscljo) [reply api/.project]: Sorry that I forgot, I'll remove it next commit (Waiting to see if you have other comments on file to remove)

**Comment by zlataovce** [comment]
everything else seems good

**Review by zlataovce — APPROVED** [review]

File: .github/workflows/gradle-publish.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set the Build number
+        id: calculated_build_number
+        env:
+          NUM: ${{ github.run_number }}
+        run: echo ::set-output name=build_n::$(($NUM+865))
```
Comment (Misat11) [inline .github/workflows/gradle-publish.yml:17]: I guess you have taken this file from BedWars itself. This paet is just for manually changing the build number. I think you can just ommit that step and in the build step itself you can just change the build number variable to ${{ github.run_number }}. Alternatively you can just change 865 to the last version of sba built by jenkins (but I don't remember it)

Comment (boiscljo) [reply .github/workflows/gradle-publish.yml:17]: I did actually took it from bedwars, but ai donMt even think I use that number anywhere in the build. If it was used, I would have put 308(last sucessful build on jenkins for SBA)

**Pull request closed without merging**